infrateam_first_responder.md

```diff
@@ -129,7 +127,8 @@ Other places to monitor:
   - `#sdr-operations`
   - `#sul-cap-collab` - has developers in the School of Medicine working on the Profiles project (which we connect to with our sul-pub system)
   - `#web-archiving`
-* Resque dashboards  e.g.
+* Queue dashboards:
+  - **Note** that you may want to clear old failed jobs at the beginning of your first responder shift so you have a clean slate to monitor, but do this deliberately and with caution: clearing failed jobs may prevent investigating current known first responder issues.
```

Comment:
Caution here.
robots - we have a sweeper that will rerun stuck jobs (I think?  or only waiting?) and we have workflows that should have the info ... but I'm still a bit unclear if we can just toast all of them without worry.
preservation - I would recommend AGAINST clearing these.  Retrying all connection errors and then clearing those, yes.  Possibly retrying non-connection errors and clearing them is okay.
was-registrar - dunno.
dor-services - we have the procedure for some stuff in the hand off doc - maybe move that here?   Not sure we can/should just "clear" other stuff.

Reply:
I'm going to remove it altogether.